sql/src/test/java/io/crate/integrationtests/DDLIntegrationTest.java

```diff
@@ -329,6 +330,33 @@ public void testCreateTableWithCompositeIndex() throws Exception {
         assertEquals(1L, response.rowCount());
     }
 
+    @Test
+    public void test_create_table_with_check_fail_on_insert() {
+        execute("create table t (id integer primary key, qty integer, constraint check_1 check (qty > 0))");
+        execute("insert into t(id, qty) values(0, null), (1, 1)");
+        refresh();
+        execute("select id, qty from t order by id");
+        assertEquals(2, response.rowCount());
+        assertEquals(printedTable(response.rows()), "0| NULL\n1| 1\n");
```

Comment:
```suggestion
        execute("select id, qty from t order by id");
        assertEquals(printedTable(response.rows()),
            "0| NULL\n" +
            "1| 1\n"
        );
```

The `rowCount == 2` is implied by the `printedTable` assertion.